.github/workflows/build.yaml

```diff
@@ -4,18 +4,39 @@ on:
   schedule:
     # “At 00:00 (UTC) on Sunday.”
     - cron: '0 0 * * 0'
-  pull_request:
   push:
-    branches:
-      - master
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        package: [flutter_simple_treeview, linked_scroll_controller, scrollable_positioned_list, self_storing_input, visibility_detector]
+        version: [stable, master]
+        exclude:
+          - package: visibility_detector
+            version: stable
```

Comment:
wonder what this does exactly. If the pubspec constraints for that package is higher than the current stable, it'll just fail pub get?